install/kubernetes/helm/istio/values.yaml

```diff
@@ -232,6 +232,9 @@ mixer:
   replicaCount: 1
   imagePullPolicy: IfNotPresent
   resources: {}
+  # TODO: Enable livenessProbe and readinessProbe in 0.9.0.
+  livenessProbeEnabled: true
+  readinessProbeEnabled: false
```

Comment:
Not sure liveness enabled and readiness enable is right for 0.8

The reverse may be better if you want one. I believe we enable readiness for pilot.

Reply:
Both of these should and will be false. I only enabled it for CircleCI tests.